src/Keboola/StorageApi/Metadata.php

```diff
@@ -22,85 +22,81 @@ public function __construct(Client $client)
     }
 
     /**
-     * @param $bucketId
+     * @param string $bucketId
      * @return array list of metadata
      */
-    public function listBucketMetadata($bucketId)
+    public function listBucketMetadata($bucketId): array
     {
-        /** @var array $result */
         $result = $this->client->apiGet("buckets/{$bucketId}/metadata");
+        assert(is_array($result));
         return $result;
     }
 
     /**
-     * @param $tableId
+     * @param string $tableId
      * @return array list of metadata
      */
-    public function listTableMetadata($tableId)
+    public function listTableMetadata($tableId): array
     {
-        /** @var array $result */
         $result = $this->client->apiGet("tables/{$tableId}/metadata");
+        assert(is_array($result));
         return $result;
     }
 
     /**
-     * @param $columnId  -- ex: "in.c-bucket.table.column"
+     * @param string $columnId  -- ex: "in.c-bucket.table.column"
      * @return array list of metadata
      */
-    public function listColumnMetadata($columnId)
+    public function listColumnMetadata($columnId): array
     {
-        /** @var array $result */
         $result = $this->client->apiGet("columns/{$columnId}/metadata");
+        assert(is_array($result));
         return $result;
     }
 
     /**
-     * @param $bucketId
-     * @param $metadataId
-     * @return mixed|string
+     * @param string $bucketId
+     * @param string|int $metadataId
      */
-    public function deleteBucketMetadata($bucketId, $metadataId)
+    public function deleteBucketMetadata($bucketId, $metadataId): void
     {
-        return $this->client->apiDelete("buckets/{$bucketId}/metadata/{$metadataId}");
+        $this->client->apiDelete("buckets/{$bucketId}/metadata/{$metadataId}");
```

Comment:
proc z tech DELETE requestu nic nevracime?

Reply:
1) protoze to vraci hlavicku `204 No Content` a prazdny string `''`
2) ze spousty DELETE requestu nic nevracime, takze to neni nic divnyho ;-)